en/SetupView/Sensors.md

```diff
@@ -1,42 +1,118 @@
 # Sensors
-Sensor setup allows you to configure and calibrate the sensors on your vehicle.
 
-![](../../images/setup/Sensors.jpg)
-*Note: The image shown is from a vehicle running ArduPilot firmware. PX4 Firmware options will be slightly different.*
+The *Sensor Setup* section allows you to configure and calibrate the vehicle's compass, gyroscope, accelerometer and other sensors (if present).
 
-To start the individual calibration steps click on a sensor's button. Sensors marked with red require calibration prior to flight. Sensors marked with green indicate a good calibration.
+Available sensors are displayed as a list of buttons beside the sidebar. Sensors marked with green are already calibrated. Sensors marked with red require calibration prior to flight. 
 
-## Accelerometer
-To calibrate the accelerometers on your vehicle you will be asked to place you vehicle in an orientation and hold it there until asked to move to the next step.
+![Sensors Setup screen for VTOL vehicle](../../images/setup/sensors_px4_vtol.jpg)
+
+> **Note** The image shown is from a VTOL vehicle running PX4 firmware. Other autopilot firmware and vehicles will offer slightly different options.
+
+Click on the button for each sensor to start its calibration sequence.
+
+<span id="flight_controller_orientation"></span>
+## Flight Controller Orientation (Sensor Settings / Set Orientations)
+
+> **Tip** You can skip this section if the flight controller and compass are mounted in the default orientation (upright on the vehicle and facing the front - `ROTATION_NONE`).
+
+If the autopilot/compass are mounted in any other way you will need to specify the orientations using either the **Sensor Settings** or **Set Orientation** buttons. In either case, choose the orientation for your autopilot/sensors from the selection list(s) and then press **OK**.
+
+<img src="../../images/setup/sensor_orientation_sensor_settings.jpg" style="width: 600px;"/>
```

Comment:
@DonLakeFlyer 

- What decides which option you get - Sensor Settings or Set Orientation? 
- If you don't get option to set  (external) compass orientation (Sensor settings) how are those set?